### ISSUE TITLE
alt text reminder

### ISSUE THREAD

**Issue by breadguyyy** [syuilo/Misskey#14564]
### Summary

add an option to warn when media is missing alt text 

### Purpose

helps to remind people to add alt text to media

### Do you want to implement this feature yourself?

- [ ] Yes, I will implement this by myself and send a pull request

**Comment by tai-cha** [comment]
Do you have any ideas on how to do so, without degrading the user experience?

**Comment by breadguyyy** [comment]
make it a setting to enable the reminder and just have it be a modal that pops up when you click post like "one or more attachments is missing alt text" with a "back" button and a "post anyway" button

**Comment by KisaragiEffective** [comment]
I wouldn't like it to be appeared.
It is nonsense to most people.
I know this may be useful, but I don't care in general situation unless the post targets people who have to use some software to getting information from the post.

**Comment by breadguyyy** [comment]
alt text is useful for people who can't see well and use screen readers, yes. it would just be an optional setting to warn people that they're about to post media without alt text
